packages/dx-react-chart/src/chart.tsx

```diff
@@ -16,7 +16,15 @@ import { Palette } from './plugins/palette';
 import { Root } from './templates/layout';
 import { Label } from './templates/label';
 
-class RawChart extends React.PureComponent {
+const defaultProps = {
+  height: 500,
+};
+
+type RawChartDefaultProps = Readonly<typeof defaultProps>;
+
+class RawChart extends React.PureComponent<RawChartProps & RawChartDefaultProps> {
```

Comment:
It should be `.PureComponent<RawChartProps>`.
`RawChartDefaultProps` is a subset of `RawChartProps` and should already be contained in the latter.

Reply:
There are several variants. If I use  `.PureComponent<RawChartProps>` then I should add `!` to the `height` property in the code. Sometimes it brings a lot of trouble.  The hack with `RawChartDefaultProps` solve that problem. Look the [article](https://medium.com/@martin_hotell/react-typescript-and-defaultprops-dilemma-ca7f81c661c7) where I have find that possibility.

Reply:
At the moment, it works because withComponents returns 'any' type. Once it fixed, this will not work because of incompatible props types: 
`export const Chart: React.ComponentType<RawChartProps> = withComponents({ Root })(RawChart);`